### ISSUE TITLE
Próxima actualización campaña #YoMeVacuno el lunes 9 de agosto

### ISSUE THREAD

**Issue by MinCiencia** [MinCiencia/Datos-COVID19#1052]
Hola, para incorporar los nuevos esquemas de vacunación, se harán cambios en los sistemas de reporte y de acceso a datos, por lo que se suspenden actualizaciones hasta el lunes.

Gracias.

**Comment by paarriagadap** [comment]
Muy buena noticia la de estos nuevos esquemas de vacunación de agosto ✌🏼

**Comment by Jorgecibia** [comment]
Alguna informacion de cuando se publicaran datos, gracias ?

**Comment by dna33** [comment]
Hola, se acaban de actualizar.

**Comment by paarriagadap** [comment]
Hola, pretenden mantener así los nombres de los fabricantes de vacunas?
![image](https://user-images.githubusercontent.com/63430031/128890670-782b3278-d581-4da7-8e32-1c2cced37315.png)
Porque seguro que les cambia a varios lo que tenían guardado o programado. Probablemente lo mejor será dejar como antes, con nombre de fabricante que está entre paréntesis no más.

**Comment by MinCiencia** [comment]
Buen punto, el procesamiennto conserva el nombre que den en DEIS al dato, hubo cambios?

**Comment by paarriagadap** [comment]
Sí, ahora quedaron todos como salen en la foto, antes todos tenían sólo el nombre entre paréntesis. Parece que CanSino está distinto ahora precisamente por un cambio que se hizo cuando se inició su vacunación hace semanas (decía Campaña SARS-COV...)

**Comment by AntoineBraultChile** [comment]
Hola, vamos a tener los datos sobre las dosis de refuerzo ? Muchas gracias.

**Comment by MinCiencia** [comment]
Hola, @AntoineBraultChile. Así es, esos datos estarán dentro de los productos de la campaña.
